### PR TITLE
added notice

### PR DESCRIPTION
**Pull request by coin96**

### CODE REVIEW

**Comment by chain518** [comment]
I think features related to pool are not correct
it needs to check those parts again

File: README.md [hunk 1]
```diff
@@ -202,3 +202,8 @@ $ forge test
 The primary license for JOJO smart contract EVM is the Business Source License 1.1 (BUSL-1.1), see LICENSE. Minus the following exceptions:
 
 - Some script have a GPL license
+
+## Author
```
Comment (chain518) [inline README.md:206]: thank you for your kindness

**Comment by universal9622** [comment]
> I think features related to pool are not correct it needs to check those parts again

i agree

File: README.md [hunk 2]
```diff
@@ -202,3 +202,8 @@ $ forge test
 The primary license for JOJO smart contract EVM is the Business Source License 1.1 (BUSL-1.1), see LICENSE. Minus the following exceptions:
```
Comment (universal9622) [inline README.md:202]: it was great team work
thanks

Comment (coin96) [reply README.md:202]: me too
thanks

File: README.md (hunk 1 above)
Comment (coin96) [reply README.md:206]: me too
thanks

**Comment by coin96** [comment]
ok sure
i wll check it again
thank you

**Pull request closed without merging**